app/config/keymaps.js

```diff
@@ -1,4 +1,5 @@
 const {readFileSync} = require('fs');
+const {normalize} = require('../../lib/utils/keymaps-normalize');
```

Comment:
For now, no file located in`lib/` directory is required from `app/**`.

It could be better to put this `normalize()` function in `app/utils/`